pkg/piecestore/rpc/server/ttl/ttl.go

```diff
@@ -0,0 +1,126 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package ttl
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+	"time"
+
+	_ "github.com/mattn/go-sqlite3" // sqlite is weird and needs underscore
```

Comment:
it's because we're using the import side effects. when the code is imported, there is code that runs in an init() function that we want to run and register with the database/sql package, so database/sql knows how to use sqlite3

Reply:
👍